hw9/main.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
+
+void translateArray(int numbers[], int size) {
+    for (int i = 0; i < size; ++i) {
+        if (numbers[i] > 0) {
+            numbers[i] *= 2;
+            std::cout << numbers[i] << " ";
+        } else {
+            numbers[i] = 0;
+            std::cout << numbers[i] << " ";
+        }
+    }
+}
```

Comment:
functions shouldn't talk. please, create printArray function and you can call it before and after this function calling

Reply:
Tnx